src/app/shared/money-input/money-input.component.css

```diff
@@ -0,0 +1,32 @@
+input {
+  text-align: center;
+  background: #ffffff;
```

Comment:
Use css-variables to not duplicate values

Reply:
done